build.rs

```diff
@@ -1,4 +1,18 @@
-use std::io::Cursor;
+use std::fs;
+use serde::Deserialize;
+
+/// Struct representation of the manifest file drivers.json
+#[derive(Deserialize)]
+struct DriverManifest {
+    #[serde(alias = "acquire-driver-common")]
+    common: String,
+    #[serde(alias = "acquire-driver-zarr")]
+    zarr: String,
+    #[serde(alias = "acquire-driver-egrabber")]
+    egrabber: String,
+    #[serde(alias = "acquire-driver-hdcam")]
+    hdcam: String,
+}
```

Comment:
Consider using [rename_all](https://serde.rs/container-attrs.html) here

It makes the fields a bit wordier, but the association with the data is a little clearer in the code imo. It also makes the first better with automatic code gen later (via macros), which might be a thing if the set of drivers expands.

```suggestion
#[derive(Deserialize)]
#[serde(rename_all = "kebap-case")]
struct DriverManifest {
    acquire_driver_common: String,
    acquire_driver_zarr: String,
    acquire_driver_egrabber: String,
    acquire_driver_hdcam: String,
}
```